source/direct_debit/index.html.md.erb

```diff
@@ -5,285 +5,253 @@ weight: 80
 
 # Direct Debit
 
-Before reading this section, you should be familiar with [making
-payments](/making_payments). You should also
-have [integrated your service](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api) with the GOV.UK Pay API.
+You can use Direct Debit if you want to take recurring payments.
 
-## Synchronous and asynchronous payment methods
+You can take payments whenever you need to. You can take the same amount or a different amount each time.
 
-Card payments are a synchronous payment method. When a user
-completes a card payment using GOV.UK Pay, the platform immediately confirms
-to your service whether the payment has succeeded or failed. If a user pays by card, you can immediately provide the purchased service to the user.
+You cannot automatically schedule payments. If you need to take regular payments, you could create your own scheduler to send API calls on a set schedule.
 
-Direct Debit payments are an asynchronous payment method. When a user
-completes a Direct Debit payment process using GOV.UK Pay, the platform makes
-a request to the user’s bank to set up a Direct Debit mandate and transfer the
-funds. It takes 3 to 4 working days to confirm the success of the payment.
+## Before you start
 
-## Get started
-
-To use the Direct Debit feature, you [must request a GOV.UK Pay
+To use the Direct Debit feature, you must [request a GOV.UK Pay
 Direct Debit test account](/support_contact_and_more_information/#contact-us) from our support team.
 
-[Contact us](/support_contact_and_more_information/#contact-us) if you want to add Direct Debit to an existing integration.
-
-### Set up an account with the Direct Debit supplier
+You must also set up an account with our Direct Debit supplier, GoCardless.
```

Comment:
I’m worried someone could see this and go off to GoCardless’s website and try to set up an account. They don’t need to do this. They can set up an account as part of the account linking flow. Can we phrase this so it informs them they’ll need to set this up as part of the set-up process but not in a way that it could be interpreted as an instruction to follow now?

Reply:
I could remove 'GoCardless', so it wouldn't be possible to read this sentence as a direct instruction? That would be closer to the original heading (which was '## Set up an account with the Direct Debit supplier'.)